Gemini/cfg/cfg.php

```diff
@@ -9,4 +9,25 @@
         'user' => 'changeme',
         'password' => 'changeme',
     ],
+
+    // Valid log levels:
+    //  - DEBUG
+    //  - INFO
+    //  - NOTICE
+    //  - WARNING
+    //  - ERROR
+    //  - CRITICAL
+    //  - ALERT
+    //  - EMERGENCY
+    //  - NONE
+    // if none is used the log file won't be opened.
+    'loglevel' => 'NONE',
+    'logfile' => '../gemini.log',
+
+    // Toggles JWT security for the service.
+    'security enabled' => true,
+    // Path to the syn config file for authentication.
+    // Example can be found here:
+    // https://github.com/Islandora-CLAW/Syn/blob/master/conf/syn-settings.example.xml
+    'security config' => '../syn-settings.xml',
```

Comment:
Also based on how far down that file was being called, we should probably recommend the use of absolute file paths.